extensions/ql-vscode/src/view/variant-analysis/VariantAnalysisSkippedRepositoryRow.tsx

```diff
@@ -0,0 +1,48 @@
+import { VSCodeBadge, VSCodeCheckbox } from '@vscode/webview-ui-toolkit/react';
+import * as React from 'react';
+import styled from 'styled-components';
+import { WarningIcon } from '../common';
+import { VariantAnalysisSkippedRepository as SkippedRepo } from '../../remote-queries/shared/variant-analysis';
+
+export type VariantAnalysisSkippedRepositoryRowProps = {
+  repository: SkippedRepo,
+};
+
+const Row = styled.div`
+  display: flex;
+  flex-direction: row;
+  gap: 0.5em;
+  align-items: center;
+`;
+
+const ChevronIcon = styled.span`
+  color: var(--vscode-descriptionForeground);
+`;
+
+const PrivacyText = styled.span`
+  font-size: small;
+  color: var(--vscode-descriptionForeground);
+`;
+
+function getPrivacyElement(isPrivate: boolean | undefined) {
+  if (isPrivate === undefined) {
+    return undefined;
+  }
+  const text = isPrivate ? 'private' : 'public';
+  return <PrivacyText>{text}</PrivacyText>;
+}
+
+export const VariantAnalysisSkippedRepositoryRow = ({
+  repository,
+}: VariantAnalysisSkippedRepositoryRowProps) => {
+  return (
+    <Row>
+      <VSCodeCheckbox />
+      <ChevronIcon className="codicon codicon-chevron-right" />
```

Comment:
If we're using the `Codicon` component, this would change to:

```suggestion
      <ChevronIcon name="chevron-right" label="Expand" />
```

Reply:
I'm not really sure what to put as the label, because it doesn't actually expand. As far as I can tell from the designs, the chevron is there to make it look like the other pages, but there's no case where it could expand and show more for this view. It can certainly be "expand" for now though. Would also be good for this to match whatever it used in the scanned repos tab.